### ISSUE TITLE
sgf replay can't get game.get_winner() 

### ISSUE THREAD

**Issue by sugizo** [atw1020/sente#27]
tested on colab and jupyter (localhost)
```
def random_player(game):
    move = random.choice(list(game.get_legal_moves() ) )
    return move

def play_game(player1, player2):
    game = sente.Game(9, sente.rules.JAPANESE)
    try:
        while not game.is_over():
            move = random_player(game)
            game.play(move)
            clear_output(wait = True)
            print(game)
    except KeyboardInterrupt:
        msg = "Game interrupted!"
        return (None, msg, game)
    file_sgf = "sgf/sente-random_player.sgf"
    sgf.dump(game, file_sgf)
    print('Winner :', game.get_winner() )

def play_sgf(file_sgf):
    game = sgf.load(file_sgf)
    try:
        for move in game.get_default_sequence():
            game.play(move)
            clear_output(wait = True)
            print(game)
    except KeyboardInterrupt:
        msg = "Game interrupted!"
        return (None, msg, game)
    print('File :', file_sgf)
    print('Winner :', game.get_winner() )
```

then run to record the game into *.sgf
```
play_game(random_player, random_player)
```

result you can see the **winner**

edit the "sgf/sente-random_player.sgf" using temporary solution on [issue/1](https://github.com/atw1020/sente/issues/26)

and to replay previous game recorded
```
play_sgf("sgf/sente-random_player.sgf")
```

**result not expected**
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-52-8fbc0def84c5> in <module>
----> 1 play_sgf("sgf/sente-random_player.sgf", pause = 0.0)

<ipython-input-51-e2defd67b697> in play_sgf(file_sgf, pause)
---> 37     print('Winner :', game.get_winner() )
     38 

ValueError: could not get results of game, the game is still ongoing
```

temporary solution, to get the winner or result is by extract the metadata, 
but what's going on, 
the function of play_sgf() is to replay the *.sgf file, then expected result is must be same when playing it using play_game() function

best regards

**Comment by atw1020** [comment]
Hi Sugizo,

Could you send me an SGF that you have generated? I think this is a bug but I want to see the game that sente is reading.

**Comment by sugizo** [comment]
generated "sgf/sente-random_player.sgf" (the problem one)
```
(;FF[4]GM[1]CA[UTF-8]
RE[W+R]
SZ[9]
RU[Japanese]
;B[ih];W[de];B[fc];W[];B[dh];W[ca];B[ad];W[eb];B[fa];W[ie];B[ii];W[bc];B[gi];W[ge];B[ag];W[ef];B[af];W[ae];B[ga];W[gd];B[di];W[ab];B[cg];W[dd];B[hh];W[bf];B[ha];W[ic];B[fh];W[cd];B[fd];W[cf];B[];W[gh];B)
```

after try temporary solution still get the same result
1 remove on the last B) into )
2 add empty barckets on the last B[]

**Comment by atw1020** [comment]
This is a bug with resignation. Unfortunately, I'm currently traveling and can't fix it until I get home. I'll let you know when I've fixed it

**Comment by atw1020** [comment]
Bug fixed in 0.3.0
